MyPortfolio.services.api/Brokers/DateTime/DateTimeBroker.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using System;
+
+namespace MyPortfolio.Services.Api.Brokers.DateTime
```

Comment:
The folder and namespace should be plural.

Reply:
Named Folder name as "Brokers" and I observe the same in namespace. Please correct me if I understood wrongly.